client/src/redux/PostsRedux.js

```diff
@@ -97,3 +104,16 @@ export const loadSinglePostRequest = id => {
     }
   };
 };
+
+export const addPostRequest = post => {
+  return async dispatch => {
+    dispatch(startRequest());
+    try {
+      let res = await axios.post(`${API_URL}/posts`, post);
```

Comment:
zmiennej res w tym przypadku nie używasz to nie ma potrzeby jej deklarować

Reply:
też zauważyłam (wyświetliło mi się), ale jest w zadaniu, że ma być :/

Reply:
wróć, bez tego nie działa (serio!) dodawanie postu

Reply:
Bez zmiennej?? Co to znaczy nie działa? Ta zmienna nic tu nie robi

Reply:
nie dodaje się post kiedy zakomentuję.

Reply:
Ale jakie zakomentuje;-) sama zmienna nie jest potrzebne,  ale odwołanie do api jest potrzebne , po prostu zostaw 

await axios.post(`${API_URL}/posts`, post);

Reply:
Ach, bo ja przed kawą :D

Reply:
Ja też:-)

Reply:
☕️ 😈